LightAPRS-hab/LightAPRS-hab.ino

```diff
@@ -86,6 +98,12 @@ void setup() {
   pinMode(BattPin, INPUT);
   pinMode(PIN_DRA_TX,INPUT);
 
+  pinMode(RedLedPin, OUTPUT);     // red LED - turn on once when APRS is powered on
+  pinMode(YlwLedPin, OUTPUT);     // yellow LED - turn on when GPS fix, turn off when no GPS fix
+  pinMode(GrnLedPin, OUTPUT);    // green LED - turn on when transmitting, turn off when not
+
+  RedLedON;
```

Comment:
Could be nice to toggle the Yellow and Green LEDs on for a second or two on startup so we can confirm that they're wired correctly